apps-rendering/src/client/callouts.ts

```diff
@@ -0,0 +1,209 @@
+import type { FormField } from '@guardian/apps-rendering-api-models/formField';
+import type { FormOption } from '@guardian/apps-rendering-api-models/formOption';
+import { ArticlePillar, ArticleSpecial } from '@guardian/libs';
+import type { ArticleFormat, ArticleTheme } from '@guardian/libs';
+import { withDefault } from '@guardian/types';
+import { ElementKind } from 'bodyElementKind';
+import { pipe, resultFromNullable } from 'lib';
+import {
+	andThen,
+	arrayParser,
+	booleanParser,
+	documentFragmentParser,
+	fieldParser,
+	map,
+	map2,
+	map7,
+	maybe,
+	numberParser,
+	parse,
+	stringParser,
+	succeed,
+} from 'parser';
+import type { Parser } from 'parser';
+import { createElement as h } from 'react';
+import ReactDOM from 'react-dom';
+import { Result } from 'result';
+import CalloutComponent from '../components/Callout';
+
+// **** Hydration **** \\
+
+type Callout = {
+	kind: ElementKind.Callout;
+	isNonCollapsible: boolean;
+	heading: string;
+	formId: number;
+	formFields: FormField[];
+	name: string;
+	description?: DocumentFragment;
+	activeUntil?: number;
```

Comment:
this isn't quite the same as the `type Callout` defined in [`bodyElement.ts`](https://github.com/guardian/dotcom-rendering/blob/da07d20e4bb9698dab7d93900afdbf0fdf9cf0a8/apps-rendering/src/bodyElement.ts#L129) - should they match do you think? 

I also wondered if it's worth exporting that type from `bodyElement.ts`, but I'm not certain they should be coupled in that way?